system-tests/universe/test-universe.js

```diff
@@ -186,11 +186,18 @@ describe("Universe", function() {
 
     // Confirm the deletion
     cy.get(".modal.modal-small input").type(serviceName);
-    cy.get(".modal.modal-small").contains("Delete").click();
+    cy
+      .get(".modal.modal-small button.button-danger", {
+        timeout: Timeouts.ANIMATION_TIMEOUT
+      })
+      .contains("Delete Service", { timeout: Timeouts.ANIMATION_TIMEOUT })
+      .click();
 
     cy
-      .get(".page-body-content table")
-      .contains(serviceName)
+      .get(".page-body-content table", {
+        timeout: Timeouts.SERVICE_DEPLOYMENT_TIMEOUT
```

Comment:
are all these `timeout` necessary? I only see the app time running increasing.

Reply:
This doesn't mean that it will wait for x seconds and then do the action. This means that is will try to do that for x seconds until it fails. We have this for every deployment. Sometimes they are 1 second but sometimes they are 10 seconds.
This makes is more flaky-proof than slow.